relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -322,13 +319,24 @@ impl TransactionExtractor<'_> {
             }
         }
 
+        // Internal usage counter
+        metrics.project_metrics.push(
+            TransactionMetric::Usage {
+                tags: UsageTags {
+                    has_profile: self.has_profile,
+                },
+            }
+            .into_metric(timestamp),
+        );
+
         // Duration
+        let duration = relay_common::time::chrono_to_positive_millis(end - start);
         metrics.project_metrics.push(
             TransactionMetric::Duration {
                 unit: DurationUnit::MilliSecond,
-                value: relay_common::time::chrono_to_positive_millis(end - start),
+                value: duration,
                 tags: TransactionDurationTags {
-                    has_profile: self.has_profile,
+                    has_profile: self.has_profile, // TODO: Stop producing this in v3
```

Comment:
Are we still producing this tag until we update the consumers?

Reply:
We have to.

Reply:
There's no functional need to remove the tag, everything works correctly with and without the tag, but as @jjbayer pointed out we need to keep it until the consumers are fully updated.